modules/dailymotionBidAdapter.js

```diff
@@ -1,11 +1,43 @@
 import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { ortbConverter } from '../libraries/ortbConverter/converter.js';
 import { VIDEO } from '../src/mediaTypes.js';
 import { deepAccess } from '../src/utils.js';
 import { config } from '../src/config.js';
 import { userSync } from '../src/userSync.js';
 
 const DAILYMOTION_VENDOR_ID = 573;
 
+const dailymotionOrtbConverter = ortbConverter({
+  context: {
+    netRevenue: false,
+    ttl: 30,
```

Comment:
Required configuration for the bid Response : https://github.com/prebid/Prebid.js/blob/master/libraries/ortbConverter/README.md

Need to retrieve the correct one